config/config.py

```diff
@@ -18,3 +20,21 @@
 csv_file_suffix = ".csv"
 json_file_suffix = ".json"
 json_asff_file_suffix = ".asff.json"
+config_yaml = "providers/aws/config.yaml"
+
+
+def change_config_var(variable, value):
+    with open(config_yaml) as f:
+        doc = yaml.safe_load(f)
+
+    doc[variable] = value
+
+    with open(config_yaml, "w") as f:
+        yaml.dump(doc, f)
```

Comment:
Why do we need this? I think this config file must be read only.

Reply:
I think that we have to have the same functionality as version 2, at least at the beginning. That's why the best option for keeping both feature is to do it this way.